### PR TITLE
added data from hygraph & updated gitignore

### PR DESCRIPTION
**Pull request by JustinLung**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,91 +1,62 @@
...
-      description: "Lekkere Italiaanse Pasta",
-      time: 90
-    },
-  ]
+export default function Home({ workouts, meals }: any) {
   return (
     <>
       <CardSlider cardSectionHeading="Featured Workouts" cardSectionLink="/workouts">
-        {workouts.map((workout) => {
+        {workouts.length === 0 && (
```
Comment (boudewijnbout) [inline src/pages/index.tsx:10]: You use && but there is only one conditional your checking?

**Review by boudewijnbout — CHANGES_REQUESTED** [review]

File: src/lib/queries.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        workoutTitle
+        workoutTime
+        workoutImage {
+          url
+        }
+      workoutDescription
+    }}`;
+
+export const mealsQuery = `query Assets {
+      meals {
```
[the remark this answers is not in the document]
Comment (JustinLung) [reply src/lib/queries.ts:12]: fixed the tabbing

File: src/pages/index.tsx (hunk 1 above)
Comment (JustinLung) [reply src/pages/index.tsx]: fixed the conditional rendering

**Review by boudewijnbout — APPROVED** [review]
LGTM!

**Pull request merged**